reporting/pom.xml

```diff
@@ -62,6 +62,11 @@
             <groupId>com.github.spullara.mustache.java</groupId>
             <artifactId>compiler</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${apache.commons.lang.version}</version>
```

Comment:
a child-pom should not mention the dependency versions. It need to inherit from root pom.